dpgen/generator/arginfo.py

```diff
@@ -194,10 +221,12 @@ def model_devi_lmp_args() -> List[Argument]:
                  doc=doc_model_devi_f_avg_relative),
         Argument("model_devi_clean_traj", [
                  bool, int], optional=True, default=True , doc=doc_model_devi_clean_traj),
-        Argument("model_devi_merge_traj", [
-                 bool], optional=True, default=False , doc=doc_model_devi_merge_traj),
+        Argument("model_devi_merge_traj", 
+                 bool, optional=True, default=False , doc=doc_model_devi_merge_traj),
         Argument("model_devi_nopbc", bool, optional=True, default=False,
                  doc=doc_model_devi_nopbc),
+        Argument("model_devi_plumed", 
```

Comment:
`model_devi_plumed_path` can be inserted here

https://github.com/deepmodeling/dpgen/blob/5ae6b2f70d191e20ba0794bed2ef03d9e4162f1f/dpgen/generator/run.py#L972-L973